src/main/java/bit/javaoop/CEO.java

```diff
@@ -0,0 +1,42 @@
+package bit.javaoop;
+
+import static bit.javaoop.Company.companyManagers;
+
+public class CEO extends AbstractEmployee {
+    private Manager[] managers = new Manager[companyManagers];
+    private int totalManagers = 0;
+
+    public CEO(String name, double salary) {
+        super(name, salary);
+    }
+
+    public Manager[] getManagers() {
+        return managers;
```

Comment:
Jak robimy getter na kolekcję czy tablicę to dobrze jest zwrócić kopię naszej kolekcji/tablicy np. 
`Arrays.copyOf(managers,managers.length);` w przypadku tablicy
`new ArrayList<>(mojaLista)` w przypadku listy